editor/src/components/canvas/controls/parent-outlines.tsx

```diff
@@ -82,7 +86,9 @@ export const ParentOutlines = controlForStrategyMemoized(
         }
         const isSlotTarget =
           findMaybeConditionalExpression(targetParent, store.editor.jsxMetadata) != null
-        const target = isSlotTarget ? EP.parentPath(targetParent) : targetParent
+        const target = isSlotTarget
+          ? getActualReparentParentForConditional(targetParent, store.editor.jsxMetadata)
```

Comment:
I didn't catch this before! what is this isSlotTarget behavior here? is this walking up to find the first non-conditional parent? so it can get the size of it for the outline? in that case I think the function's name is misleading because the _actual_ reparent target _is_ the conditional expression

Reply:
I would just call it `findFirstNonConditionalAncestor`

Reply:
ah, good point :D